README.md

```diff
@@ -169,6 +169,107 @@ package doesn't currently support the functionality that you need.
 
 
 
+
+# A Worked Example
```

Comment:
I wonder if we should have this extended example in the docs, broken up into section so that we can show the output at each stage, and explain what is going on.

And then a shorter, much simpler example in the Readme

Reply:
I think you're right. Don't think it's in scope for this PR though. Should be done in the future.